app/assets/stylesheets/cards.css.scss

```diff
@@ -0,0 +1,61 @@
+
+$FLICKR_PINK: #ff2e92;
+$FLICKR_BLUE: #007bdc;
+
+$loader-size: 30px; 
+
+$loader-gutter: $loader-size/10;
+$loader-transform: $loader-size + $loader-gutter * 2;
+
+.loader {
+  position: relative;
```

Comment:
Properties should be ordered background, border, border-radius, height, margin, position, width